src/components/panel/panel.js

```diff
@@ -2054,6 +2064,14 @@ MdPanelRef.prototype.removeFromGroup = function(groupName) {
   }
 };
 
+/**
+ * Possible default closeReasons for the close function.
+ * @enum {string}
+ */
+MdPanelRef.closeReasons = {
```

Comment:
You'll want this to be exported on the mdPanel service as well so users can access them. (Follow what the other enums did.)